groups/tests/test_forms.py

```diff
@@ -36,10 +37,11 @@ def test_form_fields(self):
         self.assertCountEqual(fields, expected)
 
     def test_form_valid(self):
-        file = factories.FileCommentFactory.create().file.file
-        data = {'file': file}
+        f = uploadable_file()
+        f.name = 'image.png'
```

Comment:
Oh, whoops.  It already is, but I made this change while experimenting on the way to doing it in `incuna-test-utils`. Thanks!